static/js/apps/diff/app.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useRef, useState } from "react";
+
+const INIT_URL = [
+  "https://datacommons.org",
+  "https://autopush.datacommons.org",
+  "",
+];
+
+function splitUrl(url: string) {
+  const urlObj = new URL(url);
+  const domain = `${urlObj.protocol}//${urlObj.host}`;
+  const rest = `${urlObj.pathname}${urlObj.search}${urlObj.hash}`;
+  return { domain, rest };
+}
+
+export function App(): JSX.Element {
+  const ref = useRef<HTMLInputElement>(null);
+  const iframeRef1 = useRef<HTMLIFrameElement>(null);
+  const iframeRef2 = useRef<HTMLIFrameElement>(null);
+  const [url, setUrl] = useState<string[]>(INIT_URL);
+  const [tmpUrl, setTmpUrl] = useState<string[]>(INIT_URL);
```

Comment:
nit: inputUrl? to be more specific about the use of this state

Reply:
Done